nixopsaws/resources/ec2_rds_dbinstance.py

```diff
@@ -91,7 +104,14 @@ def resource_id(self):
 
     def create_after(self, resources, defn):
         return {r for r in resources if
-                isinstance(r, nixopsaws.resources.ec2_rds_dbsecurity_group.EC2RDSDbSecurityGroupState)}
+                isinstance(r, nixopsaws.resources.ec2_rds_dbsecurity_group.EC2RDSDbSecurityGroupState)
+                or isinstance(r, nixopsaws.resources.ec2_rds_subnet_group.EC2RDSSubnetGroupState)
+                or isinstance(r, nixopsaws.resources.ec2_security_group.EC2SecurityGroupState)}
+
+    def destroy_before(self, resources):
```

Comment:
Do we really need `destroy_before` ? afaik it's automatically handled by doing the opposite of create_after

Reply:
I didn't know this, as I was exploring the solution for #11